docs/podtemplates.md

```diff
@@ -57,8 +57,12 @@ The current fields supported are:
   to use when dispatching the Pod. This can be used when workloads of specific types need specific schedulers,
   e.g.: If you are using volcano.sh for Machine Learning Workloads, you can pass the schedulerName and have Tasks be 
   dispatched by the volcano.sh scheduler.
+<<<<<<< HEAD
```

Comment:
It looks like something went wrong in the merge here.

Reply:
woops! Seems like i messed to add the merged file. Resolved now.